### PR TITLE
Don't exclude community facilities from drug stock

### PR DESCRIPTION
**Pull request by prabhanshuguptagit**
**Story card:** [ch4143](https://app.clubhouse.io/simpledotorg/story/4143/don-t-exclude-community-facilities-from-drug-stock)

## Because

We exclude community sized facilities for drug stock currently. Following a recent discussion, we have decided to include all facilities for drug stock.

## This addresses

Removes community size facility exclusion.

### CODE REVIEW

File: app/views/api/v3/analytics/user_analytics/show.html.erb [hunk 1]
```diff
@@ -110,16 +110,14 @@
...
-    <% unless current_facility.community? %>
-      <div class="card">
-          <h3>Drug stock report</h3>
-          <p style="text-align: left; padding: 6px 0;">Please submit your monthly hypertension drug stock report for end of <%= @for_end_of_month.to_s(:mon_year) %></p>
-          <%= link_to "Submit Drug Stock", new_webview_drug_stock_url(user_id: current_user.id,
-            access_token: current_user.access_token,
-            facility_id: current_facility.id), class: "button" %>
-      </div>
-    <% end %>
+    <% if current_facility_group.region.feature_enabled?(:drug_stocks) %>
```
Comment (harimohanraj89) [inline app/views/api/v3/analytics/user_analytics/show.html.erb:113]: 👾

**Review by harimohanraj89 — APPROVED** [review]
Looks clean, I guess we have tests to clean up.

File: app/views/api/v3/analytics/user_analytics/show.html.erb (hunk 1 above)
Comment (prabhanshuguptagit) [reply app/views/api/v3/analytics/user_analytics/show.html.erb:113]: can't find, what is it? 
https://media.giphy.com/media/XdDsb6psTES0eQo8Vg/giphy.gif

Comment (harimohanraj89) [reply app/views/api/v3/analytics/user_analytics/show.html.erb:113]: Sorry, my bad 😅  I think this line is indented one level too deep.

**Comment by harimohanraj89** [comment]
:shipit:

**Pull request merged**